turbopack/crates/turbo-tasks-backend/src/backend/operation/invalidate.rs

```diff
@@ -126,6 +176,12 @@ pub fn make_task_dirty_internal(
         }
         _ => unreachable!(),
     };
+    let _span = tracing::trace_span!(
+        "make task dirty",
+        name = ctx.get_task_description(task_id),
+        cause = cause.to_string()
+    )
+    .entered();
```

Comment:
Review to myself: It would be better to not cover the `schedule` call below to avoid nesting here